tests/Unit/Helpers/GovukPage/ConfirmTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace AnthonyEdmonds\GovukLaravel\Tests\Unit\Helpers\GovukPage;
+
+use AnthonyEdmonds\GovukLaravel\Helpers\GovukPage;
+use AnthonyEdmonds\GovukLaravel\Pages\Page;
+use AnthonyEdmonds\GovukLaravel\Tests\TestCase;
+
+class ConfirmTest extends TestCase
+{
+    protected array $page;
+
+    protected function makePage(
```

Comment:
Methods should be ordered by their visibility - public at the top, then protected, then private at the bottom.